apps/portals/src/configurations/crc-researcher/synapseConfigs/hidden.ts

```diff
@@ -26,7 +26,6 @@ export const hidden: SynapseConfig = {
         onClick: (event) => {
           handleParticipantWorkflowChange(event, 'Uncategorized')
         },
-        isRowSelectionSupported: true,
```

Comment:
Removed isRowSelectionSupported from props.  Now all custom commands are assumed to support either case (when row selections are available or unavailable)